examples/03_connectivity/plot_rest_parcellations.py

```diff
@@ -0,0 +1,194 @@
+"""
+Clustering methods to learn a brain parcellation from rest fMRI
+====================================================================
+
+We use spatially-constrained Ward-clustering and KMeans to create a set
+of parcels.
+
+In a high dimensional regime, these methods can be interesting
+to create a 'compressed' representation of the data, replacing the data
+in the fMRI images by mean signals on the parcellation, which can
+subsequently be used for statistical analysis or machine learning.
+
+Also, these methods can be interesting to learn functional connectomes
+based on these parcellations and be able to used in a classification
```

Comment:
'Also, these methods can used to learn functional connectomes and  subsequently for classification tasks...''